lib/application-processing/resolvers.ts

```diff
@@ -956,7 +970,8 @@ export const updateApplicationProcessingUploadDocuments: Resolver<
       data: {
         applicationProcessing: {
           update: {
-            documentsUrl,
+            documentsUrl: signedUrl,
+            documentsS3ObjectKey: documentsUrl,
```

Comment:
The `documentsS3ObjectKey` should be added in the gql input schema and then saved here. `documentsUrl` will not exist on in the table after the schema changes so it can be removed

Ik it's a bit confusing but remember that S3ObjectKey != S3Url
```suggestion
            documentsS3ObjectKey,
```